SDPi_Supplement/asciidoc/volume2/dev-25/tf2-dev-25.adoc

```diff
@@ -0,0 +1,67 @@
+[#vol2-clause-dev-25]
+=== Discover BICEPS Services [DEV-25]
+
+==== Scope
+[sdpi_transaction#transaction-dev-25,actors='actor-somds-provider actor-somds-consumer',figure=vol2-figure-dev-25-sequence,messages='message-discover-biceps-services']
+
+This transaction is used to exchange the resources metadata between a <<actor_somds_provider>> and a <<actor-somds-consumer>>.
+
+==== Actor Roles
+
+.Actor Roles DEV-25
+[cols="1,2"]
+|===
+|Actor |Roles
+
+|<<actor-somds-consumer>>
+| Asks for the resource representation of the <<actor_somds_provider>> via the <<message-discover-biceps-services>> messages.
+
+|<<actor_somds_provider>>
+| Sends its resource representation to the <<actor-somds-consumer>> via the <<message-response-discover-biceps-services>> messages.
+
+|===
+
+==== Referenced Standards
+
+* <<ieee-11073-10207-2017>> Section 9 Discovery Model
+
+==== Messages
+.Message Interaction Diagram DEV-25
+[plantuml#vol2-figure-dev-25-sequence, target=puml-dev-25-sequence, format=svg, reftext='{figure-caption} {counter:refnum}',width=100%]
+....
+include::../../plantuml/vol2-figure-dev-25-sequence.puml[]
+....
+
+[#message-discover-biceps-services]
+===== TransferGet
+The TransferGet message is used for a <<actor-somds-consumer>> to ask for the resource representation data for a <<actor_somds_provider>>.
+
+====== Trigger Events
+This message is sent if a <<actor-somds-consumer>> has an endpoint reference and transport address for a <<actor_somds_provider>>.

```

Comment:
```suggestion
This message is sent when a <<actor-somds-consumer>> has discovered a <<property_transport_address>> for a <<actor_somds_provider>>.
```

You should merge current master into your branch and watch out for `SDPi_Supplement\asciidoc\volume0\tf0-ch-d-glossary.adoc`. There you can append <<property_transport_address>>, which I will then also use in DEV-23 and DEV-24.

Proposed description:

```asciidoc
[[property_transport_address]] Transport Address: A physical endpoint address that can be used to communicate with a <<actor_somds_provider>>.
```